### PR TITLE
[Android] Application.Current.Exit

### PR DESCRIPTION
**Pull request by pkar70**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the current behavior?
 No support of this method.

## What is the new behavior?
 Method supported on Android.

## PR Checklist

Please check if your PR fulfills the following requirements:
- [X] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [X] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Comment by pkar70** [comment]
CI failed with 'out of memory' - could it be restarted?

File: src/Uno.UI/UI/Xaml/Application.cs [hunk 1]
```diff
@@ -53,14 +53,20 @@ public ApplicationTheme RequestedTheme
...
-
-		[NotImplemented]
+		public event UnhandledExceptionEventHandler UnhandledException;
+
+#if __ANDROID__
+		public void Exit()
+		{
+			Android.OS.Process.KillProcess(Android.OS.Process.MyPid());
+		}
+#else
```
Comment (jeromelaban) [inline src/Uno.UI/UI/Xaml/Application.cs]: Please move the Android specific code to the `Application.Android.cs` file

Comment (pkar70) [reply src/Uno.UI/UI/Xaml/Application.cs]: But what with [unimplemented] for all other platforms? It is defined in Application.cs here, not in Generated ("// Skipping already declared method Windows.UI.Xaml.Application.Exit()") - I don't know why. Should I move Exit to Generated file?

Comment (jeromelaban) [reply src/Uno.UI/UI/Xaml/Application.cs]: Use `#if !__ANDROID__` for the not implemented method in this file.

**Comment by pkar70** [comment]
As far as I remember, Visual Studio complains about "mixed line endings" and convered all line endings to... I don't know, but consistent in file.

**Comment by jeromelaban** [comment]
That command most likely changed the whitespaces as well, and this will hide the attributions.

**Comment by pkar70** [comment]
But any opening/editing of this file, under Visual Studio, will convert line endings.
I can open it in github web interface, copy whole file from current version, re-apply changes - but I don't know what will be effect of this.

windiff correctly shows difference in only changed lines, VisualStudio also.

**Comment by jeromelaban** [comment]
The file is considered as changed completely, resetting the file to its original state is more appropriate. You should also consider squashing your commits as I mentioned in a previous PR.

**Comment by pkar70** [comment]
Ok, I will try to reset file using web interface, to skip VisualStudio's line endings conversion.

**Comment by pkar70** [comment]
Same effect.

**Comment by jeromelaban** [comment]
The file is still completely modified. Please recreate your changes so only your changes appear in the modified content.

**Comment by pkar70** [comment]
But I don't know how to recreate changes - every method I tried, it ends with same problem.
The only other method I can imagine is to close this PR, create new branch, reapply changes (but manually, and not as 'cherrypick'), and create new PR.

**Comment by jeromelaban** [comment]
Reset your branch to master `git reset origin/master --hard`, re-create the update, then do `git push -f`, it will override your changes here.

**Comment by pkar70** [comment]
Done as you wrote.

**Pull request merged**